src/cascadia/TerminalControl/TermControl.h

```diff
@@ -164,8 +167,11 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         Windows::UI::Xaml::DispatcherTimer _autoScrollTimer;
         std::optional<std::chrono::high_resolution_clock::time_point> _lastAutoScrollUpdateTime;
 
+        winrt::Windows::UI::Composition::ScalarKeyFrameAnimation _bellLightAnimation;

```

Comment:
Not sure about `_bellLightAnimation` living here - I want it to be _somewhere_ so we don't need to keep recreating the animation every time we want to have a bell flash, but at the same time its wasteful creating this if the bellStyle is only audio or none

Reply:
meh that seems fine to me. In the future, we could always init this as 

```c++
winrt::Windows::UI::Composition::ScalarKeyFrameAnimation _bellLightAnimation{ nullptr };
```

and only ctor it when actually requested in the settings, but again, _meh_